addon/components/g-map-infowindow.js

```diff
@@ -43,10 +44,30 @@ const GMapInfowindowComponent = Ember.Component.extend({
     }
   },
 
+  optionsChanged: observer('disableAutoPan', 'maxWidth', 'pixelOffset', function() {
```

Comment:
I like that those options are being set as individual attributes for this component, however it makes it a bit inconsistent with how custom options are being set in `{{g-map}}` component (via `options` attr).

Probably it's ok for now, but I think I'll update `{{g-map}}` to this approach as well in the future.